src/components/sort-by-select/SortBySelect.jsx

```diff
@@ -0,0 +1,37 @@
+import { useTranslation } from 'react-i18next'
+
+import MenuItem from '@mui/material/MenuItem'
+import Select from '@mui/material/Select'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+
+import { styles } from '~/components/sort-by-select/SortBySelect.styles'
+
+const SortBySelect = ({ setSortBy, sortBy, sortingFields }) => {
```

Comment:
if i want to use select for filter, not for sorting, do possibility to path title by props

Reply:
thought about it, thx

Reply:
@MickhaelSh mb it will be need to change name from SortBySelect to smthng like AppFilterSelect or just AppSelect?

Reply:
AppSelect would be nice

Reply:
would be it good if select can be without title so that by default prop of title will have empty string and by condition there will be no ":" symbol and box will have " display:'block' "? @MickhaelSh

Reply:
@MickhaelSh , changed names and added title in props